src/Symfony/Bundle/FrameworkBundle/Resources/config/cache.xml

```diff
@@ -59,20 +55,26 @@
         </service>
 
         <service id="cache.adapter.psr6" class="Symfony\Component\Cache\Adapter\ProxyAdapter" abstract="true">
+            <tag name="cache.pool" provider="cache.default_psr6_provider" clearer="cache.default_pools_clearer" />
             <argument /> <!-- PSR-6 provider service -->
             <argument /> <!-- namespace -->
             <argument /> <!-- default lifetime -->
         </service>
 
-        <service id="cache.adapter.redis" class="Symfony\Component\Cache\Adapter\RedisAdapter" abstract="true">
+        <service id="cache.adapter.redis" class="Symfony\Component\Cache\Adapter\RedisAdapter" abstract="true" lazy="true">
```

Comment:
I just made `cache.adapter.redis` lazy so that no connection is opened unless needed.